test/data/test_functional.py

```diff
@@ -74,6 +83,43 @@ def test_sentencepiece_tokenizer(self):
         self.assertEqual(list(spm_generator([test_sample]))[0],
                          ref_results)
 
+    def test_BasicEnglishNormalize(self):
```

Comment:
Do you have a test that tries to jit this?

Reply:
I actually simplified my test case and added the jit test which is fully working. Once the PR fixing the string split lands, I will make my test case more complicated.